src/ArcGISRuntime.Samples.Shared/Models/SampleModel.cs

```diff
@@ -141,6 +143,22 @@ public enum SampleType
         public DirectoryInfo SampleFolder { get; set; }
 
         /// <summary>
+        /// Gets the (expected) assembly-qualified name for the VB sample
+        /// </summary>
+        [IgnoreDataMember]
+        public String SampleVbTypeName
```

Comment:
Without the summary comment of the property I wouldn't know what SampleVbTypeName was for